drivers/hmis/lib/tasks/one_time_migration_20241111.rake

```diff
@@ -0,0 +1,70 @@
+desc 'One time data migration to populate created_by and updated_by on Custom Assessments'
+# rails driver:hmis:migrate_assessments_20241111
+task migrate_assessments_20241111: [:environment] do
```

Comment:
This could use a more descriptive name, `migrate_assessment_user_references_` or something? We already have an assessment migration job so just to reduce confusion.